deployment/config.go

```diff
@@ -79,6 +79,18 @@ type Config struct {
 	TerraformStateDir string `default:"/var/lib/mattermost-load-test-ng" validate:"notempty"`
 	// URI of an S3 bucket whose contents are copied to the bucket created in the deployment
 	S3BucketDumpURI string `default:"" validate:"s3uri"`
+	// An optional URI to a MM server database dump file
+	// to be loaded before running the load-test.
+	// The file is expected to be gzip compressed.
+	// This can also point to a local file if prefixed with "file://".
+	// In such case, the dump file will be uploaded to the app servers.
+	DBDumpURI string `default:""`
+	// An optional list of IPs present in the posts from the DB dump
+	// that contain permalinks to other posts. These IPs are replaced,
+	// when ingesting the dump into the database, in every post that
+	// uses them with the public IP of the first app instance, so that
+	// the permalinks are valid in the new deployment.
+	PermalinkIPsToReplace []string `validate:"each:ip"`
```

Comment:
I haven't followed the original implementation but asking here for clarity. Does using the first app node's IP to replace have performance consequences? I'd expect either the SiteURL to be used to allow for proper load balancing or otherwise equally distribute to all the available nodes IPs.

Reply:
That is actually true. In case of HA deployments, we can use the nginx siteURL. @agarciamontoro - thoughts?

Reply:
Yeah, that's a good point. Created https://mattermost.atlassian.net/browse/MM-53348 to track this.